examples/todo/index.js

```diff
@@ -1,18 +1,27 @@
 const { Keystone } = require('@keystonejs/keystone');
-const { MongooseAdapter } = require('@keystonejs/adapter-mongoose');
+const { PrismaAdapter } = require('@keystonejs/adapter-prisma');
```

Comment:
We probs want to revert these changes before merging?

Reply:
Yep, this example will be completely reverted before the merge, this is just my scratch space right now 👍